articles/20240821_oss_model.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Surviving as Open Source These Days: Harder Than You Think"
+description: "Read along as we discuss common failiures in OSS models, and how to plan one which is perfect for your startup."
+date: 2024-08-21
+author: "Hunain Ahmed"
+tags: ["OSS", "open-source software", "cockroachdb", "licensing"]
+---
+
+# Surviving as Open Source These Days: Harder Than You Think
+
+## Overview
+
+Open-source software has been hailed as one of the forces that puts people together in innovation and democratizes access to really cool tools. More recent trends, however, show companies shifting away from full open licenses into more restrictive enterprise-focused models. The questions should be:
+- what exactly has driven changes from previously loved open source companies like Cockroach Labs and HashiCorp?
+- What does this all mean for the future of open-source software?
+- What should tech-startups learn from it?
+
+This article explores the licensing changes in prominent OSS projects, focusing on CockroachDB’s recent shift. We’ll discuss the motivations behind these decisions, identify common patterns, and provide insights into what constitutes a robust and sustainable OSS model.
+
+## Case Study
+
+### Cockroach DB
+
+The very first open-source release of CockroachDB was done under the Apache 2.0 License, after which it gained extreme adoption and huge people/community support. However, CockroachDB faced abuse by cloud-providers, so in 2019, Cockroach Labs changed its licensing to the Business Source License, defining the very notion of open access and finding a way to combine it with control in one commercially available instrument. Recently, they introduced a proprietary model in the CockroachDB software license. Although strategic, this move brought controversy to the community since it deviated from the primary values of the project—being open-sourced.
```

Comment:
do the interlinking from our sitemaps, at least 3 pages, consider adding a new definition realted to the topic: oss licence or similar